PusherMainView/PusherMainViewTests/PusherStoreSpec.swift

```diff
@@ -63,6 +69,8 @@ extension ObserverMock: PusherInteractable {
 
 final class PusherStoreSpec: QuickSpec {
     override func spec() {
+        var router = RouterMock()
+        let viewController = NSViewController()
         let fileName = "pusherfile.txt"
         let filePath = FileManager.default.currentDirectoryPath.appending("/\(fileName)")
```

Comment:
Can we use temp directory here?

Reply:
@imaginaris Can you create a ticket for this improvement?
It's not part of this PR.

Reply:
Ok.  Please check SDKCF-5520

Reply:
Sorry @malayli please ignore. I just noticed that "pusherfile.txt" is being removed after each test. There's no need to use temp directory.